dashboard/design/survey_summary_diagram.puml

```diff
@@ -0,0 +1,143 @@
+@startuml
+
+left header
+  <size:14>Last updated: 4/18/19
+  <size:14>Work in progress
+end header
+
+title
+  <color:#blue><size:18>**JotForm Survey Summary Assembly Line**
+end title
+
+start
+
+partition 1.Ingest {
```

Comment:
Note, I don't think it's a problem with this plan but what you've described doesn't capture the existing placeholder / hydration behavior with JotForm surveys that allows us to note that a submission occurred immediately, before actually receiving the contents of that submission from JotForm on a cronjob later. (I think)

Reply:
Correct, it is not in the scope of this design. For the purpose of downloading only new submissions, we can use this API `https://api.jotform.com/docs/#form-id-submissions` with submission range filter like `{"id:gt":"31974353596870"}`. (We will have to save mapping from jotform_id to last_downloaded_submission_id somewhere.)

Do you know what are other purposes of the placeholder?